ce/condor_validation.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+source ../func/messages.sh
+
+container_name="shared_volume-JCentral-dev-CE-1"
+
+# Log in to the Docker container and run the "condor_q" command.
+output1=$(sudo docker exec -it "$container_name" /bin/bash -c "condor_q")
+output2=$(sudo docker exec -it "$container_name" /bin/bash -c "condor_status")
+
+# Check if condor_q is working.
+if [[ "$output1" == *"-- Schedd: localhost.localdomain"* ]]; then
+    print_success "Success! condor_q is working."
+else
+    print_error "Error! condor_q is not working."
+fi
+
+# Check if condor_status is working.
+if [[ "$output2" == *"Name"* ]]; then
```

Comment:
Here, check whether there is an entry starting with slot and ending with worker1 like this. 
![image](https://github.com/FYP-Jalien/test-suite/assets/85403622/8aec1989-bafd-465d-9ac6-915b3008e9a2)
Can use grep to select a line with a pattern

Reply:
Addressed by https://github.com/FYP-Jalien/test-suite/pull/19/commits/37354501a55cc1e86e2b462ab83b10f27d745afd